R/cases_to_infections.R

```diff
@@ -1,58 +0,0 @@
-#' Cases to infections
```

Comment:
Current version of function not used, and not particularly useful method, but will need replacement at some point in future.

Reply:
Sure - I'll convert this to an issue.

Reply:
Converted to issue for future addition; see #15